docs/topics/feed-exports.rst

```diff
@@ -233,6 +234,13 @@ The serialization format to be used for the feed. See
 
 .. setting:: FEED_EXPORT_FIELDS
 
+FEED_EXPORT_ENCODING
+--------------------
+
+The encoding to be used for the feed. Defaults to UTF-8.
```

Comment:
How would I phrase it though? Wouldn't it be confusing to say it's
"\uXXXX escape
sequences" ?

On Tue, Jul 12, 2016 at 3:16 PM, Paul Tremberth <notifications@github.com>
wrote:

> In docs/topics/feed-exports.rst
> <https://github.com/scrapy/scrapy/pull/2034#discussion_r70435005>:
>
> > @@ -233,6 +234,13 @@ The serialization format to be used for the feed. See
> >
> >  .. setting:: FEED_EXPORT_FIELDS
> >
> > +FEED_EXPORT_ENCODING
> > +--------------------
> > +
> > +The encoding to be used for the feed. Defaults to UTF-8.
>
> This is not correct for JSON(lines), the default encoding is \uXXXX
> escape sequences.
> A special case indeed, but worth a mention I believe.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/scrapy/scrapy/pull/2034/files/df4190bfd442de7b829a06059d744491455f593c#r70435005>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AC-3KLke72tdcawwYV8ryP2LEk5d2gl3ks5qU5PCgaJpZM4Iu3Yz>
> .
>